android/build.gradle

```diff
@@ -192,3 +192,58 @@ dependencies {
         }
     }
 }
+
+def isScreensExampleApp() {
+    return project.hasProperty('isScreensExampleApp') && project.property('isScreensExampleApp')
+}
+
+task copyCodegenArtifacts(type: Copy) {
+    group 'After build tasks'
+    description 'Tasks which copy codegen artifacts to paper architecture'
+    
+    if (!isScreensExampleApp()) {
+        return
+    }
+
+    dependsOn tasks.generateCodegenArtifactsFromSchema
+
+    def absoluteCodegenArtifactsPaperDestination = "${project.rootDir}/../../${project.property('codegenArtifactsPaperDestination')}"
+    def absoluteCodegenArtifactsSource = "${project.rootDir}/../../${project.property('codegenArtifactsSource')}"
```

Comment:
These lines assume that `codegenArtifactsPaperDestination` & `codegenArtifactsSource` properties do exist. I think it would be better to extract these two to separate functions and handle missing values gracefully e.g. with nice error messages.

Reply:
And we should add a comment in code that they're defined in gradle.properties file or smth

Reply:
First thing changed in https://github.com/software-mansion/react-native-screens/pull/2168/commits/bd65c0c7bb6d6a12facbf79e40c1a04b89aebb4f

Reply:
Comment added in https://github.com/software-mansion/react-native-screens/pull/2168/commits/340de960ef878893869aa6f4c33bb9a0fdf730ac